release.sh

```diff
@@ -56,6 +56,20 @@ for platform in ${platforms}; do
 
 		echo "Building ${binary_name} v${version} for platform ${platform} / arch ${arch}..."
 		GO111MODULE=on GOOS=${platform} GOARCH=${arch} CGO_ENABLED=0 go build -ldflags "${ldflags}" -o "${release_dir}/${binary_name}" .
+		if [[ "$platform" == "darwin" && "${arch}" == "amd64" ]]; then
```

Comment:
why windows does not need the test?

Reply:
If this works for one of the builds, it should work for all of them - and also, this is expected to be run on `darwin amd64` so the other binaries would fail to run.